celery/canvas.py

```diff
@@ -2034,7 +2034,9 @@ def __init__(self, header, body=None, task='celery.chord',
                          )
         self.subtask_type = 'chord'
 
-    def __call__(self, body=None, **options):
+    def __call__(self, body=None, max_retries=None, **options):
```

Comment:
can you please share more detail & context for the proposed change?

Reply:
change nothing, bug let user know the max_retries default value changed: https://github.com/celery/celery/blob/main/celery/canvas.py#L2190

Reply:
@Nusnus can you verify this please?

Reply:
> @Nusnus can you verify this please?

@auvipy Done. See my added comment.